.travis.yml

```diff
@@ -1,10 +1,13 @@
 language: go
 
 go:
-  - '1.10.x'
+  - '1.12.x'
```

Comment:
should not we be testing on 2 latest go version?

Reply:
According to our [guide](https://github.com/src-d/guide/blob/master/engineering/conventions/go.md#supported-go-versions), applications support one latest stable Go version.